ui/src/app/pages/workspace/create-billing-account-modal.tsx

```diff
@@ -1,76 +1,410 @@
+                onChange={(v) => validatePhoneNumber(v)}/>
+            {invalidPhoneNumberInput && <div data-test-id='invalidPhoneNumber' style={{color: colors.danger}}>
+              Invalid phone number input
+            </div>}
+          </FlexColumn>
+        </FlexRow>
+        <FlexRow style={{marginTop: '20px'}}>
+          <FlexColumn style={styles.textNormal}>
+            Your contact email address
+            <TextInput
+                data-test-id='user-contact-email'
+                disabled={true}
+                style={styles.textInput}
+                value={contactEmail}/>
+          </FlexColumn>
+          <FlexColumn style={styles.textNormal}>
+            Your Researchallofus ID
+            <TextInput
+                data-test-id='user-workbench-id'
+                style={styles.textInput}
+                disabled={true}
+                value={username}/>
+          </FlexColumn>
+        </FlexRow>
+        <FlexRow style={{marginTop: '20px'}}>
+          <FlexColumn style={styles.textNormal}>
+            Your institution
+            <TextInput
+                data-test-id='user-institution'
+                style={styles.textInput}
+                disabled={true}
+                value={verifiedInstitutionalAffiliation.institutionDisplayName}/>
+          </FlexColumn>
+        </FlexRow>
+        <FlexRow style={{marginTop: '100px', justifyContent: 'flex-end'}}>
+          <Button type='secondary'
+                  style={{fontWeight: 400, padding: '0 18px', height: '40px', marginRight: '10px'}}
+                  onClick={() => onClose()}>
+            Cancel
+          </Button>
+          <Button data-test-id='next-button'
+                  type='primary'
+                  style={{fontWeight: 400, padding: '0 18px', height: '40px', width: '93px'}}
+                  disabled={!phoneNumber}
+                  onClick={() => {setCurrentStep(2); }}>
+            Next
+          </Button>
+        </FlexRow>
+      </FlexColumn>
+    </ModalFooter>}
+    {currentStep === 2 && <ModalFooter data-test-id='step-2-modal' style={{marginTop: 0, justifyContent: 'flex-start'}}>
+      <FlexColumn style={{justifyContent: 'space-evenly', width: '100%'}}>
+        <div style={styles.textHeader}>What payment method would you like to use?</div>
+        <FlexColumn style={{marginTop: '20px', width: '100%'}}>
+          <FlexRow style={{boxSizing: 'border-box', borderRadius: '8px', border: '1px solid #CCCFD4', marginBottom: '7px'}}>
+            <RadioButton data-test-id='credit-card-radio'
+                         style={{margin: '15px', height: '17px', width: '17px'}}
+                         checked={useCreditCard === true}
+                         onChange={() => setUseCreditCard(true)}/>
+            <FlexColumn style={{marginTop: '9px', marginLeft: '15px', marginBottom: '15px'}}>
+              <FlexRow>
+                <div style={styles.textHeader}>Credit Card&nbsp;&nbsp;</div>
```

Comment:
why the two spaces? Isn't the next line line-wrapped anyways?

Reply:
That is 24 hrs after that. hence I added two space
![Screen Shot 2021-08-19 at 8 41 39 PM](https://user-images.githubusercontent.com/6351731/130161425-fb903eba-f650-41cb-8e25-376e9761bc1c.png)